examples/gromacs_eessi_config.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022-2024 The Ramble Authors
+# 
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+ramble:
+  variants:
+    package_manager: eessi
+  env_vars:
+    set:
+      OMP_NUM_THREADS: '{n_threads}'
+      # Control which microarchitecture to use with EESSI
+      EESSI_SOFTWARE_SUBDIR_OVERRIDE: '{eessi_arch_dir}'
```

Comment:
We'll soon start introducing additional EESSI versions, so maybe you already want to specify `EESSI_VERSION` here too?